ext/js/background/backend.js

```diff
@@ -622,20 +626,6 @@ export class Backend {
             }
```

Comment:
Where is `cannotAdd` being used?

Reply:
Yeah but I don't see `cannotAdd` being consumed within the function scope or am I blind?

Reply:
It's possible it may be able to be removed entirely.